### PR TITLE
Bump version to 10.2.0

### PR DESCRIPTION
**Pull request by sebastienlagarde**
Bump version to 10.2.0

### CODE REVIEW

**Review by ellioman — APPROVED** [review]

**Review by phi-lira — CHANGES_REQUESTED** [review]
Missing the bump in Version.hlsl https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.core/ShaderLibrary/Version.hlsl#L2

**Review by phi-lira — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
> Missing the bump in Version.hlsl https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.core/ShaderLibrary/Version.hlsl#L2

good catch! thanks

File: com.unity.template-hd/Packages/com.unity.template.hd/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [10.2.0] - 2020-10-19
+
+### Changed 
+- Updte SRP package to 10.2.0
```
Comment (0lento) [inline com.unity.template-hd/Packages/com.unity.template.hd/CHANGELOG.md:10]: Typo here

File: com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [10.2.0] - 2020-10-19
+
+### Changed 
+- Updte SRP package to 10.2.0
```
Comment (0lento) [inline com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md:10]: Typo here

**Pull request merged**